DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -22,19 +22,20 @@
 
 """
 
-## imports
+# imports
 import os
 import stat
 import re
-from types import  StringTypes, ListType
-## from DIRAC
+from types import StringTypes, ListType
+# from DIRAC
 from DIRAC import gLogger, S_OK, S_ERROR, gConfig
 from DIRAC.Core.DISET.RequestHandler import RequestHandler
 from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getVOForGroup
 
 from DIRAC.Resources.Storage.StorageBase import StorageBase
 
-from irods import rcConnect , rcDisconnect , clientLoginWithPassword, irodsCollection, irodsOpen, getResources , getResource #pylint: disable=import-error
+from irods import rcConnect, rcDisconnect, clientLoginWithPassword  # pylint: disable=import-error
+from irods import irodsCollection, irodsOpen, getResources  # pylint: disable=import-error
```

Comment:
Why doing it with two separate import statements ?

Reply:
Too long line. Now the question is: should this file be kept or removed?

And BTW, should the lfc_dfc_copy scripts be kept?

Reply:
I had missed that comment. 
The "traditional" way is to split with a backslash no ? 

Not up to me to decide whether this script is to be kept or not (anyway, we never used that one)